packages/myst-directives/src/blockquote.ts

```diff
@@ -1,17 +1,16 @@
 import type { DirectiveSpec, DirectiveData, GenericNode } from 'myst-common';
-import { normalizeLabel } from 'myst-common';
 import type { Container } from 'myst-spec-ext';
 import classNames from 'classnames';
+import { addCommonDirectiveOptions, labelDirectiveOption } from './utils.js';
 
 export const blockquoteDirective: DirectiveSpec = {
   name: 'blockquote',
   alias: ['epigraph', 'pull-quote'],
   doc: 'Block quotes are used to indicate that the enclosed content forms an extended quotation. They may be followed by an inscription or attribution formed of a paragraph beginning with `--`, `---`, or an em-dash.',
   options: {
-    label: {
-      type: String,
-      alias: ['name'],
-    },
+    ...labelDirectiveOption('blockquote'),
+
+    // TODO: Add enumeration in future
```

Comment:
Good that we are not adding `enumerator` on places where we do not number stuff yet.

Reply:
Looking forward to being able to number everything ... !